### ISSUE TITLE
Output file log inconsistency

### ISSUE THREAD

**Issue by karthigayini** [genome-nexus/genome-nexus#606]
Hi  @inodb 

I ran genome nexus using the docker container by converting the input VCF to MAF. The output log file and MAF are attached (in text format)

1. As you can see, the log file mentions that there are 22 entries with failed annotation, whereas the MAF file shows only 12. How can I find the rest 10?
2. In the attached MAF, there are some mutations where HGSP_short is present but not the long version. Why is that?

It will be really helpful if you could answer a few more questions.

1. Is the final o/p file named "sample.genomeNexus.maf" ?
2. What does ONP mean in the variant_type column mean? How is that different from TNP?
3. How does Nexus choose transcript ID? Is there a possibility of providing a customized list of IDs to use with the tool?

[sample1.genomeNexus.maf.txt](https://github.com/genome-nexus/genome-nexus/files/8556787/sample1.genomeNexus.maf.txt)
[sample1.log.txt](https://github.com/genome-nexus/genome-nexus/files/8556788/sample1.log.txt)

**Comment by inodb** [comment]
@karthigayini Thanks for filing this!

For the first two questions. Is this something you can help with @ao508 or @leexgh ?

For the second part of the questions:

1. I believe you can choose the name of the output file with `--output-filename <OUTPUT DESTINATION>`. Let me know if that doesn't work
2. oligonucleotide polymorphism. This is four or more nucleotides (see https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)
3. Currently we only allow for the [mskcc set of canonical transcripts](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_at_mskcc_grch37.txt) or [uniprot](https://github.com/genome-nexus/genome-nexus-importer/blob/master/data/common_input/isoform_overrides_uniprot.txt). We currently don't allow user-supplied sets of isoforms but if there is an essential use case for it we can try to build it in

Hope that helps! We'll try to address the other issues in 1&2 soon

**Comment by karthigayini** [comment]
Thanks @inodb for the explanation. 
WRT ONP: in the attached MAF file, ONP is marked for a variant with 3 Nucleotide change. Can I know why?

**Comment by karthigayini** [comment]
@leexgh  thanks.

**Comment by inodb** [comment]
@karthigayini 

>WRT ONP: in the attached MAF file, ONP is marked for a variant with 3 Nucleotide change. Can I know why?

Good catch. This looks like a bug actually. It should be TNP. Filed a separate ticket for that: https://github.com/genome-nexus/genome-nexus/issues/609. Thanks!

**Comment by karthigayini** [comment]
@inodb Just one more question. In the attached MAF file, there are some entries with Hugo Symbol as "Unknown". On inspecting those regions on the UCSC genome browser, it turns out to be FGFR3. May I know why it is not annotated?

**Comment by leexgh** [comment]
@karthigayini Could you check if those annotations are failed? I believe they should have `FAILED` in `Annotation_Status` column

**Comment by leexgh** [comment]
@karthigayini Can I ask a question? What does 'N' in `Tumor_Seq_Allele2` column mean? 
<img width="997" alt="image" src="https://user-images.githubusercontent.com/16869603/168720576-7ce3f544-c345-44d5-aa3a-5e62ba7c8593.png">
